client/model.go

```diff
@@ -243,48 +243,58 @@ type TeamProjectAssignment struct {
 }
 
 type Policy struct {
-	Id                         string `json:"id"`
-	ProjectId                  string `json:"projectId"`
-	NumberOfEnvironments       int    `json:"numberOfEnvironments"`
-	NumberOfEnvironmentsTotal  int    `json:"numberOfEnvironmentsTotal"`
-	RequiresApprovalDefault    bool   `json:"requiresApprovalDefault"`
-	IncludeCostEstimation      bool   `json:"includeCostEstimation"`
-	SkipApplyWhenPlanIsEmpty   bool   `json:"skipApplyWhenPlanIsEmpty"`
-	DisableDestroyEnvironments bool   `json:"disableDestroyEnvironments"`
-	SkipRedundantDeployments   bool   `json:"skipRedundantDeployments"`
-	UpdatedBy                  string `json:"updatedBy"`
+	Id                          string `json:"id"`
+	ProjectId                   string `json:"projectId"`
+	NumberOfEnvironments        int    `json:"numberOfEnvironments"`
+	NumberOfEnvironmentsTotal   int    `json:"numberOfEnvironmentsTotal"`
+	RequiresApprovalDefault     bool   `json:"requiresApprovalDefault"`
+	IncludeCostEstimation       bool   `json:"includeCostEstimation"`
+	SkipApplyWhenPlanIsEmpty    bool   `json:"skipApplyWhenPlanIsEmpty"`
+	DisableDestroyEnvironments  bool   `json:"disableDestroyEnvironments"`
+	SkipRedundantDeployments    bool   `json:"skipRedundantDeployments"`
+	UpdatedBy                   string `json:"updatedBy"`
+	RunPullRequestPlanDefault   bool   `json:"runPullRequestPlanDefault"`
+	ContinuousDeploymentDefault bool   `json:"continuousDeploymentDefault"`
 }
 
 type PolicyUpdatePayload struct {
-	ProjectId                  string `json:"projectId"`
-	NumberOfEnvironments       int    `json:"numberOfEnvironments"`
-	NumberOfEnvironmentsTotal  int    `json:"numberOfEnvironmentsTotal"`
-	RequiresApprovalDefault    bool   `json:"requiresApprovalDefault"`
-	IncludeCostEstimation      bool   `json:"includeCostEstimation"`
-	SkipApplyWhenPlanIsEmpty   bool   `json:"skipApplyWhenPlanIsEmpty"`
-	DisableDestroyEnvironments bool   `json:"disableDestroyEnvironments"`
-	SkipRedundantDeployments   bool   `json:"skipRedundantDeployments"`
+	Id                          string `json:"id"`
```

Comment:
We don't send the `id` on the project update payload, the `id` is a part of the path

Reply:
https://github.com/env0/terraform-provider-env0/pull/164/commits/50ae45c0e3104a4cf8bedef74180d51aaf93023d